packages/manager/apps/telecom/src/app/telecom/pack/xdsl/access/statistics/statistics.controller.js

```diff
@@ -538,4 +546,100 @@ export default class XdslStatisticsCtrl {
         this.attenuation.loading = false;
       });
   }
+
+  /**
+   * Get errors statistic for the line
+   * @param {String} period Period to request :
+   *          - daily
+   *          - monthly
+   *          - preview
+   *          - weekly
+   *          - yearly
+   * @return {promise}
+   */
+  getErrorsStatistics(period) {
+    this.errors.loading = true;
+    return this.$q
+      .all({
+        hec: this.getLinesStatistics('error:hec', period),
+        fec: this.getLinesStatistics('error:fec', period),
+        crc: this.getLinesStatistics('error:crc', period),
+      })
+      .then((stats) => {
+        this.errors.haveSeries = !!(
+          stats.hec.length &&
+          stats.fec.length &&
+          stats.crc.length
+        );
+
+        this.errors.chart = new this.TucChartjsFactory(
+          angular.copy(this.PACK_XDSL_STATISTICS.chart),
```

Comment:
`Object.assign({}, this.PACK_XDSL_STATISTICS.chart)` not sufficient ?

Reply:
Sorry but I don't have the same with this.